ksql-engine/src/main/java/io/confluent/ksql/function/udf/geo/GeoDistanceKudf.java

```diff
@@ -18,17 +18,23 @@
 import com.google.common.collect.Lists;
 import io.confluent.ksql.function.KsqlFunctionException;
 import io.confluent.ksql.function.udf.Kudf;
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
+
 import java.util.List;
 
 /**
  * Compute the distance between two points on the surface of the earth, according to the Haversine
  * formula for "great circle distance". The 2 input points should be specified as (lat, lon) pairs,
  * measured in decimal degrees.
- * 
+ *
  * <p>An optional fifth parameter allows to specify either "MI" (miles) or "KM" (kilometers) as the
  * desired unit for the output measurement. Default is KM.
- *
  */
+@UdfDescription(name = "geodistance", author = "Confluent",
+    description = "Compute the distance between two points on the surface of the earth,"
+        + " according to the Haversine formula for \"great circle distance\".")
 public class GeoDistanceKudf implements Kudf {
```

Comment:
```suggestion
 public class GeoDistanceKudf  {
```
You don't need to implement `Kudf` with the new UDF framework, and then you can also remove the `evaluate` method